augmentedreality/Examples/CppArExample/CppArExample.pro

```diff
@@ -21,6 +21,19 @@ CONFIG += c++17
 
 TARGET = CppArExample
 
+lessThan(QT_MAJOR_VERSION, 6) {
+    error("This version of the ArcGIS Runtime SDK for Qt requires Qt 6.2.4")
+}
+
+equals(QT_MAJOR_VERSION, 6) {
+    lessThan(QT_MINOR_VERSION, 2) {
+        error("This version of the ArcGIS Runtime SDK for Qt requires Qt 6.2.4")
+    }
+  equals(QT_MINOR_VERSION, 2) : lessThan(QT_PATCH_VERSION, 4) {
+    error("This version of the ArcGIS Runtime SDK for Qt requires Qt 6.2.4")
+  }
+}
```

Comment:
```suggestion
lessThan(QT_MAJOR_VERSION, 6) {
    error("This version of the ArcGIS Runtime SDK for Qt requires Qt 6")
}
```

🔧 
This is fine but it will create maintenance in the future as we move up Qt versions. For now I think we should keep it simple and stick with one Qt 6 check.

Reply:
I think this will be fine when we move up Qt versions, it is checking 6.2.4 as a minimum. This would only cause problems if we were to support Qt versions lower than 6.2.4